src/Microsoft.DotNet.ScenarioTests.FileBasedApps/FileBasedAppTests.cs

```diff
@@ -0,0 +1,359 @@
+using Microsoft.DotNet.ScenarioTests.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.ScenarioTests.FileBasedApps;
+
+public class FileBasedAppTests : IClassFixture<ScenarioTestFixture>
+{
+    private readonly ScenarioTestFixture _scenarioTestInput;
+    private readonly FileBasedAppHelper _helper;
+
+    public FileBasedAppTests(ScenarioTestFixture testInput, ITestOutputHelper outputHelper)
+    {
+        if (string.IsNullOrEmpty(testInput.DotNetRoot))
+        {
+            throw new ArgumentException("sdk root must be set for file-based app tests");
+        }
+
+        _scenarioTestInput = testInput;
+        _helper = new FileBasedAppHelper(outputHelper, _scenarioTestInput.DotNetRoot, _scenarioTestInput.SdkVersion, _scenarioTestInput.BinlogDir);
+    }
+
+    [Fact]
+    [Trait("Category", "Offline")]
+    public void VerifySimpleFileBasedApp()
```

Comment:
The Offline tests should be combined together. The goal is that the scenario tests execute a wide swath of .NET functionality. Similar with the online tests.